tests/python/rest_api/test_quality_control.py

```diff
@@ -104,8 +104,13 @@ def _test_list_reports_403(self, user, task_id, **kwargs):
     def test_can_list_quality_reports(self, admin_user, quality_reports):
         parent_report = next(r for r in quality_reports if r["task_id"])
         task_id = parent_report["task_id"]
-        reports = [parent_report] + [
-            r for r in quality_reports if r["parent_id"] == parent_report["id"]
+
+        reports = [
+            r
+            for r in quality_reports
+            if r["task_id"] == task_id
+            or r["parent_id"]
+            and quality_reports[r["parent_id"]]["task_id"] == task_id
```

Comment:
```suggestion
                or r["parent_id"] and quality_reports[r["parent_id"]]["task_id"] == task_id
```

Just makes the grouping a bit clearer.

Reply:
I agree, but this is what black does.

Reply:
Right... oh well.